front-end/src/components/VolunteerApplication.jsx

```diff
@@ -1,29 +1,89 @@
-import { useState } from "react";
+import { useEffect, useState } from "react";
+import { volunteerRoles } from "../utilities/EventUtilities";
+
 import Button from "react-bootstrap/Button";
 import Form from "react-bootstrap/Form";
 import Modal from "react-bootstrap/Modal";
+import EventDetails from "../pages/EventDetails";
+import { resolvePath } from "react-router-dom";
 
+//name, email, phone, best time to contact, have you volunteered with this organization/person before, why would you like to volunteer for this event, what skills/experience do you have? which role are you interested in?
 
+function VolunteerApplication({ show, handleClose, eventID }) {
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
```

Comment:
we should probably set the default here to the users account email - it should be stored in the user useContext still

Reply:
that's a good idea, let me look at that if ican figure it out quickly